modules/ROOT/pages/create-your-first-project-with-the-engine-apis-and-maven.adoc

```diff
@@ -21,7 +21,7 @@ In order to use the client APIs, you need to add a dependency to the `bonita-cli
 [source,xml,subs="+macros"]
 ----
 <properties>
-   <bonita.bpm.version>pass:a[{varVersion}].0</bonita.bpm.version>
+   <bonita.bpm.version>pass:a[{javadocVersion}].0</bonita.bpm.version>
```

Comment:
⚠️ I suggest we introduce a dedicated `bonitaTechnicalVersion` asciidoctor attribute with value 7.12.0 to avoid this weird version concatenation and that will allow increasing the patch component of the version if needed.
See also https://github.com/bonitasoft/bonita-documentation-site/issues/72

Reply:
ah, the Pull Request has been merged prior my comment has been submitted, so I will create a new one with my proposal 😸

Reply:
Sorry i'm so fast 🤐

Reply:
no problemo 😈

Reply:
Proposal in #1490